src/main/java/org/cbioportal/model/SurvivalEvent.java

```diff
@@ -0,0 +1,61 @@
+package org.cbioportal.model;
+
+import java.util.List;
+import java.util.function.ToIntFunction;
+
+public class SurvivalEvent {
+    List<ClinicalEvent> startClinicalEventsMeta;
+    ToIntFunction<ClinicalEvent> startPositionIdentifier;
```

Comment:
I don't see the need for the extra level of abstraction with using ToIntFunction?

Reply:
Sonarcloud suggests us to use more specific function interface whenever possible. It can improve performance, type safety and readability. If you don't like the extra things introduced to our code base, I could remove it anyway.

Reply:
No I agree that the specific function interface is the right move. I was just questioning the functional abstraction itself. Why pass a function to call at all. I can elaborate in a call

Reply:
Thanks, let's connect to discuss!